### PR TITLE
bip8: Advance to Complete

### PR DESCRIPTION
**Pull request by murchandamus**
BIP8 has not had significant updates since 2021, but continues to be relevant. E.g., the BIP110 activation appears to have been modeled on BIP8, and the Speedy Trial activation of Taproot was implemented based on BIP8.

I therefore posit that BIP8 should be advanced to Complete rather being in Draft status.

cc: @shaolinfry, @luke-jr as the BIP owners, but anyone’s review is welcome.

### CODE REVIEW

**Comment by luke-jr** [comment]
RDTS is actually BIP 9 based, unfortunately. But Taproot is a valid example. I do feel like BIP 8 could still use further elaboration (eg, explicitly laying out how the community can reject bad proposals),  but I suppose as a spec it's sufficient.

**Comment by murchandamus** [comment]
> RDTS is actually BIP 9 based, unfortunately. But Taproot is a valid example. I do feel like BIP 8 could still use further elaboration (eg, explicitly laying out how the community can reject bad proposals), but I suppose as a spec it's sufficient.

Well, a mix really. RDTS uses a `starttime` like BIP9, but otherwise height-based parameters like BIP8 and a mandatory activation like BIP8’s LOT.

Would I read too much into your comment if I were to interpret it as an ACK?

**Review by jonatack — COMMENTED** [review]
> the BIP110 activation appears to have been modeled on BIP8, and the Speedy Trial activation of Taproot was implemented based on BIP8

@luke-jr should the `bip-0008/assignments` file be updated?

**Comment by jonatack** [comment]
> BIP 8 could still use further elaboration (eg, explicitly laying out how the community can reject bad proposals), but I suppose as a spec it's sufficient

Relevant part of BIP 3 about the Complete status, if helpful:

"Subsequently, the BIP’s content should only be adjusted in minor details, e.g., to improve language, clarify ambiguities, backfill omissions in the specification, add test vectors for edge cases, or address other issues discovered as the BIP is being adopted."

**Comment by murchandamus** [comment]
If you are no longer working on moving this BIP to complete, we could instead advance it to Closed. I can open the alternative PR if that is the preference.

**Comment by murchandamus** [comment]
I added a Changelog and proposed #2235 as an alternative that moves BIP8 to Closed instead.

File: bip-0008.mediawiki [hunk 1]
```diff
@@ -275,3 +276,32 @@ A living list of deployment proposals can be found [[bip-0008/assignments.mediaw
 
 This document is dual licensed as BSD 3-clause, and Creative Commons CC0 1.0 Universal.
 
+==Changelog==
+* **1.0.0** (2026-08-03):
+** Advance to Complete.
+* **0.1.2** (2021-02-17):
```
Comment (jonatack) [inline bip-0008.mediawiki]:
```suggestion
* **0.3.2** (2021-02-17):
```

Comment (murchandamus) [reply bip-0008.mediawiki]: Thanks, fixed

**Comment by jonatack** [comment]
Per BIP author https://github.com/bitcoin/bips/pull/2163#issuecomment-4502936903, "I suppose as a spec it's sufficient."

Per BIP author https://github.com/bitcoin/bips/pull/2235#issuecomment-5172846510, "Deployed would make more sense."

As Complete is the step between Draft and Deployed, Complete is acceptable.

Possible follow-up: https://github.com/bitcoin/bips/pull/2163#pullrequestreview-4338010670.
